dist/READMEopenshifttechpreview.md

```diff
@@ -28,23 +34,34 @@ openshift_use_openshift_sdn='False'
 When the cluster install is complete and the cluster is up there will
 be no cluster networking.
 
+
 Clone ovn-kubernetes on a convenient host where you can run ansible-playbook:
 ```
 # git clone https://github.com/openvswitch/ovn-kubernetes
 # cd ovn-kubernetes/dist/ansible
 ```
 
-Edit the hosts file adding the name of the cluster master.
-
-Optionally, edit the name of the desired image into the daemonset
-yaml files. All of the daemonsets use the same image.
-The default is the community image.
+Edit the hosts file adding the name of the cluster master and select the ovn_image
+and ovn_image_pull_policy. The default is the community image.
 
 Provision the cluster for OVN:
 ```
 # ./run-playbook
 ```
 
+OVN may be removed from the cluster by running:
+```
+# ./run-playbook uninstall
+```
+
+The yaml/{ovnkube.yaml,ovnkube-master.yaml} files are now created as follows:
+```
+# cd ../images
+# make daemonsets
```

Comment:
I think it is better to make it something like 'make daemonsetyaml' to make it obvious.

Reply:
@shettyg I made your suggested changes, PTAL

Reply:
@shettyg @rajatchopra @dcbw more cleanups PTAL